tool/truffle/jruby_truffle_runner/lib/jruby_truffle_runner.rb

```diff
@@ -0,0 +1,270 @@
+require 'optparse'
```

Comment:
Anything more substantial than a simple configuration file must begin with our standard copyright header - so the JRuby people are clear what we're putting into their repository if nothing else.